README.md

```diff
@@ -1,7 +1,9 @@
 [![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+[![Build Status](https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core)
+
 # Google App Engine Plugins Core Library
 
-This is the client Java library to manage App Engine Java applications. It should be used by any application that performs App Engine Java application management. For example, the Maven, Gradle and Eclipse App Engine plugins, custom user tools, etc.
+This is the client Java library to manage App Engine Java applications. It should be used by any Java application that performs App Engine Java application management. For example, the Maven, Gradle and Eclipse App Engine plugins, custom user tools, etc.
```

Comment:
I think you need to be clear about what the purpose of this library is.

If your plan is to support feature requests and breaks reported by other users that are using this library for their own purposes, then it's public. If that's not the plan, then it's private, and you should not make/imply any support guarantees.

I'd recommend leaving this as private. Unless it is clear how/why this is valuable for other users and unless it's requested, I'd opt not to expose an API, because then you have to support it.